### PR TITLE
Fix disappearing holos

### PR DESCRIPTION
**Pull request by sarahsturgeon**

### CODE REVIEW

File: moon/autorun/server/cfc_leftover_chips.moon [hunk 1]
```diff
@@ -29,13 +30,15 @@ onCreate = (ent) ->
...
 cleanup = ->
     for leftover in pairs leftovers
         if IsValid leftover
-            property = rawget checkProperty, leftover\GetClass!
-            continue if IsValid leftover[property]
+            getter = rawget getOwner, leftover\GetClass!
+            continue unless getter
+            continue if IsValid getter leftover
 
         SafeRemoveEntityDelayed leftover, 0
```
Comment (legokidlogan) [inline moon/autorun/server/cfc_leftover_chips.moon:41]:
```suggestion
    timer.Simple 1, ->
        for leftover in pairs leftovers
            if IsValid leftover
                getter = rawget getOwner, leftover\GetClass!
                continue unless getter
                continue if IsValid getter leftover

            SafeRemoveEntityDelayed leftover, 0
```

Comment (sarahsturgeon) [reply moon/autorun/server/cfc_leftover_chips.moon:41]: The timer's actually on the hook.Add at the bottom, but I realized it needs to be 1s not 0

File: moon/autorun/server/cfc_leftover_chips.moon [hunk 2]
```diff
@@ -29,13 +30,15 @@ onCreate = (ent) ->
...
-hook.Add "PlayerDisconnected", "CFC_Leftovers_Cleanup", cleanup
+
+hook.Add "OnEntityCreated", "CFC_Leftovers_Track", onCreate
+hook.Add "PlayerDisconnected", "CFC_Leftovers_Cleanup", timer.Simple 1, cleanup
```
Comment (plally) [inline moon/autorun/server/cfc_leftover_chips.moon]:
```suggestion
hook.Add "PlayerDisconnected", "CFC_Leftovers_Cleanup", -> timer.Simple 1, cleanup
```

**Comment by legokidlogan** [comment]
Tested and working

**Pull request merged**